projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -1322,7 +1322,7 @@ public function process_submission() {
 				'post_parent'  => $post ? (int) $post->ID : 0,
 				'post_title'   => addslashes( wp_kses( $feedback_title, array() ) ),
 				// phpcs:ignore WordPress.NamingConventions.ValidVariableName.InterpolatedVariableNotSnakeCase, WordPress.PHP.NoSilencedErrors.Discouraged, WordPress.PHP.DevelopmentFunctions.error_log_print_r
-				'post_content' => addslashes( wp_kses( "$comment_content\n<!--more-->\nAUTHOR: {$comment_author}\nAUTHOR EMAIL: {$comment_author_email}\nAUTHOR URL: {$comment_author_url}\nSUBJECT: {$subject}\nIP: {$comment_author_IP}\n" . @print_r( $all_values, true ), array() ) ), // so that search will pick up this data
+				'post_content' => addslashes( wp_kses( "$comment_content\n<!--more-->\nAUTHOR: {$comment_author}\nAUTHOR EMAIL: {$comment_author_email}\nAUTHOR URL: {$comment_author_url}\nSUBJECT: {$subject}\nIP: {$comment_author_IP}\nJSON_DATA\n" . @wp_json_encode( $all_values, true ), array() ) ), // so that search will pick up this data
```

Comment:
Would be nice if we could take the _encoding_ process out to its own method where we can (more readably) do whatever we need to. Get rid of the `@` and properly check for errors, extract `\nJSON_DATA` to a class constant are some of the things that come to mind. No blocker, just a nit

Reply:
One for a future PR, I think. I want the json change to be a drop-in replacement for print_r. While it's probably ok to remove the @ I think it should remain for now.